src/app/js/formats/identifier-badge/IdentifierBadgeAdmin.js

```diff
@@ -53,7 +54,13 @@ class IdentifierBadgeAdmin extends Component {
 
     render() {
         const { p: polyglot, args: { colors, typid } } = this.props;
-
+        const items = Object.keys(resolvers).map(resolverID => (
+            <MenuItem
+                key={`resolver_${resolverID}`}
+                value={resolverID}
+                primaryText={polyglot.t(resolverID)}
```

Comment:
It seems that no DOI, DOAI nor PMID key exist in translations.tsv.

Reply:
Maybe it's not useful, as the key is the translation, and as it was already used on lines 69-70 in old version.